src/Controls/src/Core/ScrollView.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Maui.Controls
 {
 	[ContentProperty(nameof(Content))]
-	public partial class ScrollView : Layout, IScrollViewController, IElementConfiguration<ScrollView>, IFlowDirectionController
+	public partial class ScrollView : Compatibility.Layout, IScrollViewController, IElementConfiguration<ScrollView>, IFlowDirectionController
```

Comment:
Do we have missing features on the new layout that the scroll view needs?

Reply:
ScrollView probably shouldn't even be a layout. But until we have time to rewrite the cross-platform ScrollView bits, it's a subclass of the legacy Layout class. Same with Page. We won't be porting the parts it needs to the new Layout class.